### PR TITLE
Conti docs

### PR DESCRIPTION
**Pull request by profxj**
A few docs.  Not much else.

### CODE REVIEW

File: docs/scripts.rst [hunk 1]
```diff
@@ -2,6 +2,12 @@
 linetool Scripts
 ****************
 
+There are a number of scripts, many of which are GUIs,
+provided with linetools.  As regards the GUIs we warn
+that Mac users will need to set their matplotlib to
+something other than MaxOSX. See
```
Comment (nhmc) [inline docs/scripts.rst]: typo

**Comment by nhmc** [comment]
Looks good, apart from the typo I pointed out inline. Please merge when ready.

File: docs/scripts.rst (hunk 1 above)
Comment (profxj) [inline docs/scripts.rst]: fixed

**Comment by profxj** [comment]
Merging

**Pull request merged**